src/store/year.ts

```diff
@@ -0,0 +1,29 @@
+import { ActionContext } from 'vuex';
+import { State } from '@/types';
+
+export default {
+  state: {
+    currentYear: new Date().getFullYear(),
+  },
+  getters: {
+    getPreviousYear: (state: State): number => state.currentYear - 1,
```

Comment:
The prefix `get` might be unnecessary - especially when we look at how it reads in the view (*.Vue) file - nouns like `previousYear` and `nextYear` should read better